pyvista/plotting/export_vtkjs.py

```diff
@@ -634,7 +639,10 @@ def export_plotter_vtkjs(plotter, filename, compress_arrays=False):
 
     shutil.rmtree(output_dir)
 
-    print('Finished exporting dataset to: ', sceneFileName)
+    # this must occur to avoid leaks
+    scDirs.clear()
+    for array in arrays:
+        array.SetReferenceCount(0)
```

Comment:
Contortions like this tell me that something's seriously wrong with the dev wheels. Nobody should ever have to null reference counts manually...

Reply:
Actually, it's quite likely that this was an issue outside of the dev wheels, we've just never enabled `DVTK_DEBUG_LEAKS` when building VTK and tested it against Python.  There's no hurt adding this in here.